moveit_ros/planning/CMakeLists.txt

```diff
@@ -1,27 +1,34 @@
 cmake_minimum_required(VERSION 3.10.2)
 project(moveit_ros_planning)
 
-add_compile_options(-std=c++14)
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
 
 if(NOT CMAKE_CONFIGURATION_TYPES AND NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE Release)
 endif()
 
 find_package(Boost REQUIRED)
 find_package(ament_cmake REQUIRED)
-find_package(moveit_msgs REQUIRED)
-find_package(moveit_ros_perception REQUIRED)
+# find_package(moveit_msgs REQUIRED)
```

Comment:
If possible, please keep these enabled

Reply:
Removed the comment except `moveit_ros_perception`.